geometry/bbox.cpp

```diff
@@ -0,0 +1,26 @@
+#include "geometry/bbox.hpp"
+
+#include <algorithm>
+using namespace std;
```

Comment:
Empty line between `include` and `using namespace` please

Reply:
Done